reactnative/ReactNativeTemplateApp/js/index.android.js

```diff
@@ -37,22 +37,40 @@ var {
     Navigator
 } = React;
 var forceClient = require('./react.force.net.js');
+var oauth = require('./react.force.oauth.js');
 
 var App = React.createClass({
+    getInitialState: function() {
```

Comment:
NEW: Your app needs to call authenticate before doing anything that requires authentication